src/ConnectedMode/UI/OrganizationSelection/OrganizationSelectionViewModel.cs

```diff
@@ -76,4 +80,24 @@ internal void UpdateOrganizations(AdapterResponseWithData<List<OrganizationDispl
         responseWithData.ResponseData.ForEach(AddOrganization);
         RaisePropertyChanged(nameof(NoOrganizationExists));
     }
+
+    internal async Task<bool> ValidateConnectionForOrganizationAsync(string organizationKey, string warningText)
```

Comment:
Can be made private, see other comment. The warning text does not need to be parametrized as well..

Reply:
I think it does. For usability purposes.